xsl/pretext-html.xsl

```diff
@@ -9816,7 +9828,7 @@ along with MathBook XML.  If not, see <http://www.gnu.org/licenses/>.
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/doenet-standalone-test@0.6.0/dist/style.css" />
     <script onload="onLoad()" type="module" src="https://cdn.jsdelivr.net/npm/doenet-standalone-test@0.6.0/dist/doenet-standalone.min.js"></script>
     <script>
-        <xsl:text>function onLoad() {window.renderDoenetToContainer(document.querySelector(".doenetml-applet"))}</xsl:text>
+        <xsl:text>function onLoad() {document.querySelectorAll(".doenetml-applet").forEach(e=>window.renderDoenetToContainer(e)}</xsl:text>
```

Comment:
There's something buggy (with doenet-standalone?) here where if I have multiple applets on the same page, sometimes (often) the second applet isn't rendered correctly, see:

![image](https://github.com/PreTeXtBook/pretext/assets/1559632/f2529f90-83fd-4bb5-b571-44751c286ad0)